src/main/kotlin/pl/edu/agh/gem/external/controller/InternalGroupController.kt

```diff
@@ -0,0 +1,26 @@
+package pl.edu.agh.gem.external.controller
+
+import org.springframework.http.HttpStatus.CREATED
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import pl.edu.agh.gem.external.dto.GroupAttachmentResponse
+import pl.edu.agh.gem.internal.service.GroupService
+import pl.edu.agh.gem.media.InternalApiMediaType.APPLICATION_JSON_INTERNAL_VER_1
+
+@RestController
+@RequestMapping("/internal/groups")
+class InternalGroupController(
+    val groupService: GroupService,
+) {
+    @PostMapping("/{groupId}/users/{userId}/generate", produces = [APPLICATION_JSON_INTERNAL_VER_1])
+    @ResponseStatus(CREATED)
+    fun saveAttachment(
```

Comment:
would rename to generateAttachment. (but save is ok)

Reply:
sure changed